typescript-forms/form-events.ts

```diff
@@ -0,0 +1,35 @@
+const $userName = document.querySelector('#user-name');
+const $email = document.querySelector('#user-email');
+const $message = document.querySelector('#user-message');
+
+if (!$userName || !$email || !$message) {
+  throw new Error('$userName, $email, or $message query failed.');
+}
+
+function handleFocus(event: Event): void {
+  console.log('Focus event has fired');
+  const eventTarget = event.target as HTMLInputElement;
```

Comment:
This should be `as HTMLInputElement | HTMLTextAreaElement`, since we don't know which one is going to cause this function to be fired

Reply:
Thanks! Updated!